Objects/TI4_Helpers/TI4_UnitHelper.ttslua

```diff
@@ -297,6 +347,33 @@ local _flagshipModifiers = {
             end
         end
     },
+
+    ['Quetzecoatl'] = {
+        description = "Other players cannot use SPACE CANNON against your ships in this system",
+        owner = OWNER.OPPONENT,
+        type = TYPE.MUTATE,
+        apply = function(unitAttrs)
+            for unitType, attr in pairs(unitAttrs) do
+                if attr.spaceCannon then
+                    attr.spaceCannon.dice = 0
```

Comment:
"attr.spaceCannon = nil" is better, will cause it to disappear from the roller.

Reply:
and PDS2 detection probably won't respect this with the auto-fill tool, seems ok to let the players enforce that.